docs/core/compatibility/2.0-2.1.md

```diff
@@ -9,4 +9,12 @@ If you're migrating from version 2.0 to version 2.1 of .NET Core, the breaking c
 
 ## CoreFx
 
-[!INCLUDE[Instantiate struct](~/includes/core-changes/corefx/2.1/instantiate-struct.md)]
+Index:
```

Comment:
Not sure if I like the word index. I'd either use a short introductory sentence or just remove this. And I'd remove the horizontal line from after the list.